examples/vision/melanoma_classification_with_tfrecords.py

```diff
@@ -0,0 +1,438 @@
+"""
+Title: Melanoma Classification with TFRecords
+Author: Amy MiHyun Jang
+Date created: 2020/07/29
+Last modified: 2020/07/29
+Description: Classifying skin lesion images from TFRecords for melanoma diagnosis
+"""
+"""
+# 1. Introduction + Set Up
+
+Transfer learning in computer vision is the process of using a pre-trained model on a new
```

Comment:
We have [a guide on transfer learning](https://keras.io/guides/transfer_learning/), as well as [a code example based on EfficientNet](https://keras.io/examples/vision/image_classification_efficientnet_fine_tuning/). It appears the main contribution of this example is to show how to read TFRecords with tf.data. Perhaps we could have a very streamlined version of this example under the section "quick Keras recipes", focused entirely on the TFRecord part of the workflow? "How to train a Keras model on TFRecord files"